### PR TITLE
Localize content/en/docs/reference/glossary/application-developer.md

### PR DESCRIPTION
**Pull request by taniaduggal**
fixes #38332 
Signed-off-by: Tania Duggal <103496926+taniaduggal@users.noreply.github.com>

### CODE REVIEW

**Comment by NitishKumar06** [comment]
Looks good to me!
/lgtm

**Comment by taniaduggal** [comment]
@divya-mohan0209

**Review by bishal7679 — APPROVED** [review]
Taking ref from #36101
its ready-to-merge
LGTM!

**Comment by NitishKumar06** [comment]
@taniaduggal The PR is already done with the review. Kindly be patient. I'd suggest going through [this](https://github.com/kubernetes/website/pull/36101#issuecomment-1221251909) once again.

**Comment by NitishKumar06** [comment]
> @divya-mohan0209
@taniaduggal The PR is already done with the review. Kindly be patient. I'd suggest going through [this](https://github.com/kubernetes/website/pull/36101#issuecomment-1221251909) once again.

File: content/hi/docs/reference/glossary/application-developer.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+aka:
+tags:
+  - user-type
+---
+
+एक व्यक्ति जो कुबेरनेट्स क्लस्टर में चलने वाले एप्लिकेशन लिखता है।
+
+<!--more-->
+
+एप्लिकेशन डेवलपर्स एप्लिकेशन के एक हिस्से पर ध्यान केंद्रित करते हैं। उनकी कार्य सीमा के आकार में एक उल्लेखनीय अंतर हो सकता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/application-developer.md:17]: Whar you've suggested is work streams that is significantly different than focus areas. You can accept the suggestion below OR we can work together to figure out a better way to phrase the sentence. WDYT?
```suggestion
एप्लिकेशन डेवलपर्स एप्लिकेशन के एक हिस्से पर ध्यान केंद्रित करते हैं। उनके फोकस का पैमाना आकारस्वरूप काफी भिन्न हो सकता है।
```

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Review by bishal7679 — APPROVED** [review]
/lgtm

**Pull request merged**